site/documentation/content/api/credentials/index.md

```diff
@@ -62,6 +67,16 @@ The following request payload may be used to look up the hashed password for a d
 }
 ~~~
 
+The following request payload may be used to look up *or create* `x509-cert` credentials for a device with the authentication identifier `sensor1`:
```

Comment:
that is not the authentication identifier ...

What happens if the auth-id differs from the subject DN in the certificate?

Reply:
> What happens if the auth-id differs from the subject DN in the certificate?

We could add here that the device registry must check that they match and return a 400 if they don't. 
Another option would be to remove the auth-id property if a client certificate is present in the request, with the downside that the property is not always mandatory. WDYT?

Reply:
IMHO returning 400 is the better option as it does not require any changes to the default mechanism of requesting credentials.